cime_config/machines/config_batch.xml

```diff
@@ -76,6 +76,20 @@
      </queues>
    </batch_system>
 
+   <batch_system MACH="frontier" type="slurm">
+    <batch_submit>sbatch</batch_submit>
+     <directives>
+       <directive> --core-spec=$SHELL{tpn=`./xmlquery --value MAX_TASKS_PER_NODE`; if [[ $tpn > 56 ]]; then echo "64-$tpn"|bc; else echo 8; fi; }</directive>
+     </directives>
+     <queues>
+       <queue strict="true" nodemin="1" nodemax="91" walltimemax="02:00:00" default="true">batch</queue>
+       <queue strict="true" nodemin="92" nodemax="183" walltimemax="06:00:00">batch</queue>
+       <queue strict="true" nodemin="184" nodemax="1881" walltimemax="12:00:00">batch</queue>
+       <queue strict="true" nodemin="1882" nodemax="5644" walltimemax="24:00:00">batch</queue>
+       <queue strict="true" nodemin="5645" nodemax="9408" walltimemax="24:00:00">batch</queue>
```

Comment:
These 2 lines are identical: walltimemax 24hrs. Combine into 1?

Reply:
From Frontier docs. Queue policy might change. So, keeping it for now.

Reply:
https://docs.olcf.ornl.gov/systems/frontier_user_guide.html#job-priority-by-node-count